pkg/apis/paddlepaddle/v1/types.go

```diff
@@ -0,0 +1,171 @@
+package v1
+
+import (
+	"fmt"
+
+	batchv1 "k8s.io/api/batch/v1"
+	corev1 "k8s.io/api/core/v1"
+	v1beta1 "k8s.io/api/extensions/v1beta1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	// CRDKind is the kind of K8s CRD.
+	CRDKind = "TrainingJob"
+	// CRDKindPlural is the plural of CRDKind.
+	CRDKindPlural = "trainingjobs"
+	// CRDShortName is the short name of CRD.
+	CRDShortName = "tj"
+	// CRDGroup is the name of group.
+	CRDGroup = "paddlepaddle.org"
+	// CRDVersion is the version of CRD.
+	CRDVersion = "v1"
+)
+
+// CRDName returns name of crd
+func CRDName() string {
+	return fmt.Sprintf("%s.%s", CRDKindPlural, CRDGroup)
```

Comment:
Maybe just return `CRDKindPlural + "." + CRDGroup`, which will be optimized to a compile time constant. Anyway, this is very minor, `fmt.Sprintf()` is very fast anyway.